### PR TITLE
Refactor/remove prys and commented code

### PR DESCRIPTION
**Pull request by TLamb32**
## Description

This PR is mostly for formatting and removing excess data. In it, I removed pry's and other commented out code that was not implemented. I also went through each class and accompanying parser and removed unused data, i.e. (venue link, powerplay goals, etc.)

## Type of change

- [ ] fix
- [ ] feat
- [ ] test
- [x] refactor
- [ ] docs

## Checklist

- [x] code has been self reviewed
- [x] code runs without any errors
- [ ] thorough testing has been implemented if adding feature
- [x] all tests pass

### Thanks!

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -1,33 +1,26 @@
 # This class is pulling data from games.csv (or its fixture for testing purposes)
-# We will use game_id, season, away_team_id, hone_team_id, away_goals, home_goals
+# We will use game_id, season, away_team_id, home_team_id, away_goals, home_goals
```
Comment (ttakahashi1591) [inline lib/game.rb:2]: Nice catch on the spelling of `home_team_id` here!

File: lib/game_parser.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # This class is pulling data from games.csv (or its fixture for testing purposes)
-# We will use game_id, season, away_team_id, hone_team_id, away_goals, home_goals
+# We will use game_id, season, away_team_id, home_team_id, away_goals, home_goals
```
Comment (ttakahashi1591) [inline lib/game_parser.rb:2]: Nice catch on the spelling of `home_team_id` here!

File: lib/game_parser.rb [hunk 3]
```diff
@@ -14,14 +14,11 @@ def get_game_info(games_path)
...
       season = row[:season]
-      type = row[:type]
-      date_time = row[:date_time]
       away_team_id = row[:away_team_id]
       home_team_id = row[:home_team_id]
       away_goals = row[:away_goals]
       home_goals = row[:home_goals]
-      venue = row[:venue]
-      game = Game.new(game_id, season, type, date_time, away_team_id, home_team_id, away_goals, home_goals, venue)
+      game = Game.new(game_id, season, away_team_id, home_team_id, away_goals, home_goals)
```
Comment (ttakahashi1591) [inline lib/game_parser.rb:21]: Thanks for removing the additional data points that were not necessary for the project!

File: lib/team_result_parser.rb [hunk 4]
```diff
@@ -1,8 +1,14 @@
+# This class will pull from game_teams.csv.
+# We need game_id, team_id, HoA, result, head_coach, goals, shots, tackles
+
```
Comment (ttakahashi1591) [inline lib/team_result_parser.rb:3]: Adding this description is super helpful for understanding what we will be using the `team_result_parser` for!

File: lib/team_result_parser.rb [hunk 5]
```diff
@@ -96,7 +94,7 @@ def get_win_percentage(season_games)
         end
       end
     end
-    # Add default value for hash (Hash.new(0))
+
```
Comment (ttakahashi1591) [inline lib/team_result_parser.rb]: Great catch on removing this comment!

File: lib/team_result_parser.rb [hunk 6]
```diff
@@ -117,7 +115,7 @@ def get_win_percentage(season_games)
       end
     end
     win_percent = {}
-    #.merge with hashes
+
```
Comment (ttakahashi1591) [inline lib/team_result_parser.rb]: Great catch on removing this comment!

**Review by ttakahashi1591 — APPROVED** [review]
This all looks great and I appreciate you taking the time to go through the files and support with these edits.

**Review by aevans27 — APPROVED** [review]
Thanks for cleaning up the unused code.

**Pull request merged**